### PR TITLE
Fix abord behaviour logic

### PR DESCRIPTION
**Pull request by PascalPiche**

### CODE REVIEW

File: src/com/flickity.js [hunk 1]
```diff
@@ -165,7 +165,7 @@
 				}
 				
 				App.callback(o.inited);
-			} else if (slider.find(o.cellSelector.length == 1)) {
+			} else if (slider.find(o.cellSelector).length == 1) {
```
[the remark this answers is not in the document]
Comment (PascalPiche) [reply src/com/flickity.js]: Yep good for me. will add a commit tomorow. Just found it by luck.

[the remark this answers is not in the document]
Comment (nitriques) [reply src/com/flickity.js]: Nice luck.

**Comment by nitriques** [comment]
@PascalPiche Ping, since I know this as been fixed in client code ;)

File: src/com/flickity.js [hunk 2]
```diff
@@ -165,10 +165,13 @@
...
 				
 				App.callback(o.inited);
-			} else if (slider.find(o.cellSelector.length == 1)) {
+			} else if (slider.find(o.cellSelector).length === 1) {
 				slider.addClass(o.abortedClass);
 				slider.find(o.cellSelector).addClass(o.selectedClass);
 				App.callback(o.aborted);
+			} else {
+				App.log('No flickity found or multiple flickity in the scope');
+				App.callback(o.aborted);
```
Comment (nitriques) [inline src/com/flickity.js]: Should we try to always add ALL values in the default object ?

Comment (PascalPiche) [reply src/com/flickity.js]: ?

Comment (nitriques) [reply src/com/flickity.js]: aborted is not declared in `defaultOptions`. Should we define in ?

Comment (nitriques) [reply src/com/flickity.js]: @PascalPiche  ???

Comment (nitriques) [reply src/com/flickity.js]: @PascalPiche  Wanna add a commit that does it ?

**Pull request merged**